rafs/src/metadata/mod.rs

```diff
@@ -397,34 +398,42 @@ pub struct RafsSuperConfig {
     pub is_tarfs_mode: bool,
 }
 
+#[derive(Error, Debug)]
+pub enum MergeError {
+    #[error("Inconsistent RAFS version:{0}")]
+    InconsisentFsVersion(String),
```

Comment:
Better be

```
#[error("Inconsistent RAFS Filesystem: {0}")]
nconsisentFilesystem(String),
```

Reply:
There will be a warning if doesn't have an upper camel case name.May be follows better.
```
#[error("Inconsistent RAFS Filesystem: {0}")]
NconsisentFilesystem(String),
```